src/main/java/com/barterbay/app/config/SecurityConfiguration.java

```diff
@@ -1,18 +1,29 @@
 package com.barterbay.app.config;
 
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
 //TODO BB-21
+@Configuration
```

Comment:
no need @Configuration here since in @EnableWebSecurity this annotation already contains